Representable/representable
# update small logo graphic, set default mapping radius to 0, update drive landing page

**changes**
- updated lowres logo graphic to be more accurate
- set default mapping radius to 0, rather than 25/50
- updated drive landing page to match look of current state page

**to test**
- python manage.py runserver
- check logo in navbar looks as expected
- check no errors when mapping
- check drive landing page looks good

**screenshots**
![image](https://user-images.githubusercontent.com/41943646/100031459-c8f9ea80-2d99-11eb-9632-2b6ca5e1bed1.png)
